enhancements/kubernetes-nmstate-integration.md

```diff
@@ -57,15 +57,25 @@ what this project is addressing.
 
 ## Proposal
 
-A new kubernetes-nmstate handler DaemonSet is deployed in the cluster part of the OpenShift installation.
+A new kubernetes-nmstate operator Deployment is deployed to the cluster as part of the the OpenShift
+installation, via CVO.
+
+The operator has a single CRD, called `NMstate`. When a custom resource of that type is created,
```

Comment:
"When a custom resource of that type is created"

It should have a specific name?
How we handle multiple CRs?

Reply:
I'd say that's a good idea. That we look for a specific name. There only needs to be a single CR. Right now, if you add another (with a different name), it just reconciles again.